source/math/FloatVector2D.ooc

```diff
@@ -22,7 +22,7 @@ FloatVector2D: cover {
 	x, y: Float
 	area ::= this x * this y
 	length ::= this norm
-	empty ::= !(this x > 0 && this y > 0)
+	empty ::= this y equals(0.0f) || this x equals(0.0f) || this x < 0.0f || this y < 0.0f
```

Comment:
Alright. At least change the position of `x` and `y` in the first two statements and I'm happy.